lib/tasks/email_giftcards.rake

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+namespace :email_giftcards do
+  desc 'Email customers the gift cards that they ordered'
+  task :send_emails => :environment do
+    desc 'send emails'
+
+    unique_recipients = GiftCardDetail
+                        .joins(:item, :recipient)
+                        .where(items: { refunded: false })
+                        .distinct.pluck(:recipient_id)
+
+    unique_recipients.each do |id|
+      html = '<!DOCTYPE html>' \
+              '<html>' \
+              '<head>' \
+              "  <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />" \
+              '</head>' \
+              '<body>' \
+              '<p> Thank you for supporting local chinatown businesses! As our merchants open up, you will be able to redeem the giftcards you have purchased below.</p>'
+      recipient = Contact.find_by(id: id)
+      gift_cards = GiftCardDetail.where(recipient_id: id).to_a
+      html += '<style> table {border: 1px solid black}</style>'\
+            '<table>'
+      gift_cards.each do |gc|
+        gift_card_url = 'merchant.sendchinatownlove.com/voucher/' + gc.gift_card_id
```

Comment:
@justintmckibben should this be `seller_gift_card_id`? This still confuses me about which code gets used where tbh